board/build.rs

```diff
@@ -28,6 +28,9 @@ fn main() {
                     .data(imxrt_rt::Memory::Dtcm)
                     .bss(imxrt_rt::Memory::Dtcm)
                     .uninit(imxrt_rt::Memory::Dtcm)
+                    .stack_size_env_override("THIS_WONT_BE_CONSIDERED")
+                    .stack_size_env_override("BOARD_T4_STACK")
+                    .heap_size_env_override("SOME_HEAP_SIZE_BYTES")
```

Comment:
Unless I misunderstand (which is eminently possible), maybe these aren't the best names for the environment variables?

It looks to me as if:
- should I want to override the stack size, I use `BOARD_T4_STACK` - no worries here
- should I want to override the heap size, I use ` SOME_HEAP_SIZE_BYTES` - maybe `BOARD_T4_HEAP` instead?

Reply:
Yea good call. I was only thinking about tests, and forgot folks might want to build examples with these variables. All boards now support the same `BOARD_STACK` and `BOARD_HEAP` names.